R/grid_helpers.R

```diff
@@ -102,6 +95,37 @@ predict_model <- function(split, workflow, grid, metrics, submodels = NULL) {
   tibble::as_tibble(res)
 }
 
+predict_wrapper <- function(model, new_data, type, times, subgrid = NULL) {
```

Comment:
This funciton cleans up the code where it is used (a little) and would allow us to add more hooks for future special cases.